docs/source/model_design/general_research.rst

```diff
@@ -888,9 +888,125 @@ respiratory disease.
+questions from the survey. Analytical studies explore the relationship between 
+exposures (risk factors) and the outcome. The prevalence of the outcome is 
+compared between exposed and unexposed group. It's calculated as prevalence 
+of a disease in exposed population divided by prevalence of that disease in 
+unexposed population, namely the **prevalence ratio**. The prevalence ratio 
+describes the association between the exposure and prevalence of the outcome. 
+For example, we set gender as a risk factor and smoking as the outcome, assuming 
+the prevalence of smoking among men over 15 in 2005 was 30% and 4% for women at 
+same age and year, then the prevalence ratio for smoking is 8. That's being said 
+men are 8 times more likely to be smoker than women in this population, indicating 
+a strong association between sex and prevalence of smoking.
+
+
+Sample selection
+~~~~~~~~~~~~~~~~
+
+Both the method used to select samples and response rate determine how well the 
+results can be generalized to a population as a whole. In a cross-sectional 
+study, the sample is often taken from the whole population or subgroups within 
+the population that are randomly selected and highly representative. It's very 
+important to ensure the responders are representative as well, as non-response 
+might occur in a large-scale survey. The technologies to minimise this problem 
+include telephone and mail prompting, and other incentive actions. Another 
+outstaning issue related to reponse is biased reponse, where individuals are 
+more likely to answer the questions or complete the surveys with a paticular 
+characteristic or a set of characteristics. Bias will introduce when certain 
+characteristic in questionnaire is related to probability of having a disease 
+or outcome. For example, you might observe a high response rate among elder 
+people and those unemployed by conducting a door-to-door interview for a disease. 
+The results are biased because we are accidentally assume these two groups are 
+actually experiencing higher risk of disease but ignore the fact that they are 
+more likey to be stay in home during the daytime.
+
+
+Advantages and disadvantages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Cross-sectional study is less expensive and take shorter time to conduct than a 
+longitudinal study because the subjects are not necessarily to be tracked over 
+time, therefore the analysis of the study can start immediately after the data 
+collection finished. They are powerful in health planning, understadning etiology 
+and generation of hypotheses. Researchers can collect data about their 
+hypothesized phenomenon, and also gather the information about the demographic 
+and other relevant characteristics in order to compare the finidngs from 
+different groups. In addition, cross-sectional studies do not have a concern of 
+loss to follow-up.
+
+There are some disadvantages in cross-sectional study design. It's often diffcult 
```

Comment:
'often difficult' or 'infeasible'? (This is a genuine and not a rhetorical question)